sdk/rust/rustfmt.toml

```diff
@@ -1,18 +1,18 @@
 # Merge similar crates together to avoid multiple use statements.
-imports_granularity = "Module"
+# imports_granularity = "Module"
```

Comment:
let's add a not on unstable features here as well

Reply:
Done.